docs/user/commit-message.md

```diff
@@ -17,7 +17,8 @@ Reviewers: github-username-a, github-username-b
 
 The first line will be the title of the PR created by `spr diff`, and the rest of the lines except for the `Reviewers` line will be the PR description (i.e. the content of the first comment). The GitHub users named on the `Reviewers` line will be added to the PR as reviewers.
 
-The `Test Plan` section is required to be present; `spr diff` will fail with an error if it isn't.
+The `Test Plan` section is required to be present by default; `spr diff` will fail with an error if it isn't.
+You can disable this in the [configuration](/docs/reference/configuration.md).
```

Comment:
```suggestion
You can disable this in the [configuration](../reference/configuration.md).
```

The absolute path doesn't work with mdbook (which generates https://getcord.github.io./spr). You can test it yourself, if you like: `cargo install mdbook && mdbook serve` (from the root directory of the repository)